controllers/capa_controller.go

```diff
@@ -63,29 +64,22 @@ func (r *CAPAClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	var err error
 	logger := r.Log.WithValues("namespace", req.Namespace, "cluster", req.Name)
 
+	logger.Info("Reconciling AWSCluster")
+
 	cluster := &capa.AWSCluster{}
 	if err := r.Get(ctx, req.NamespacedName, cluster); err != nil {
-		logger.Info("Cluster no longer exists")
+		if k8sErrors.IsNotFound(err) {
+			logger.Info("Cluster no longer exists")
+			return ctrl.Result{}, nil
+		}
```

Comment:
[There is a newish helper](https://github.com/giantswarm/aws-resolver-rules-operator/blob/main/controllers/resolver_rules.go#L82) to make this more concise `IgnoreNotFound`.

Reply:
done